upup/pkg/fi/assettasks/copyfile.go

```diff
@@ -92,7 +108,7 @@ func (_ *CopyFile) Render(c *fi.Context, a, e, changes *CopyFile) error {
 func transferFile(c *fi.Context, source string, target string, sourceSHA string, sourceSHALocation string) error {
```

Comment:
Rename to `copyFile`?  What are we doing here?

Reply:
Added code comments about what the func is doing. I would rather not rename it to copyfile, as it is a bad naming style to name a method the code file name.